demumble.cc

```diff
@@ -113,8 +116,9 @@ int main(int argc, char* argv[]) {
         size_t special = strcspn(cur, "_?");
         if (print_mode == kPrintAll)
           printf("%.*s", static_cast<int>(special), cur);
-        else if (need_separator)
+        else if (need_separator) {
```

Comment:
please revert this change

Reply:
Is this extra commit OK or would you like a fresh branch with this squashed?